api.py

```diff
@@ -306,3 +306,33 @@ async def get(self, request, *args, **kwargs):
         result = await search_location_by_label(query)
         response = result
         return json(response, status=200)
+
+@ns_loc_func.route('/to-DGGS')
+class to_DGGS(Resource):
+    """Function for finding an array of DGGS cells by a loci uri"""
+
+    @ns.doc('find_dggs_by_loci_uri', params=OrderedDict([
+        ("uri", {"description": "Search DGGS cells by loci uri",
+                    "required": True, "type": "string"}),
+    ]), security=None)
+    async def get(self, request, *args, **kwargs):
+        """Calls DGGS table to query DGGS cells by loci uri"""
+        print(request.args)
+        query = str(next(iter(request.args.getlist('uri'))))
+        result = await find_dggs_by_loci_uri(query)
+        return json(result, status=200)
+
+@ns_loc_func.route('/find-at-DGGS-cell')
+class find_at_DGGS_cell(Resource):
```

Comment:
It would be good to validate the format of the DGGS Cell ID string. Currently any string can be submitted as an input. 
Perhaps add a regex to check that this string is something like `S3006887558`?

Reply:
The basic API looks good. Could we return the data structure to align with the other API return structures?
```
{
    "meta": {
        "count": 5,
        "offset": 0,
        "dggs_cell_id": "S3006887558",
    },
    "locations" : [
        {
            "uri": "http://linked.data.gov.au/dataset/asgs2016/statisticalarealevel1/60302107209",
            "datatypeURI" : "http://linked.data.gov.au/def/asgs#StatisticalAreaLevel1",
            "datatype": "asgs16_sa1"
        },
        {
            "uri": "http://linked.data.gov.au/dataset/asgs2016/statisticalarealevel2/603021072",
            "datatypeURI" : "http://linked.data.gov.au/def/asgs#StatisticalAreaLevel2",
            "datatype": "asgs16_sa2"
        },
        {
            "uri": "http://linked.data.gov.au/dataset/asgs2016/statisticalarealevel3/60302",
            "datatypeURI" : "http://linked.data.gov.au/def/asgs#StatisticalAreaLevel3",
            "datatype": "asgs16_sa3"
        },
        {
            "uri": "http://linked.data.gov.au/dataset/asgs2016/localgovernmentarea/63010",
            "datatypeURI" : "http://linked.data.gov.au/def/asgs#LocalGovernmentArea",
            "datatype": "asgs16_lga"
        },
        {
            "uri": "http://linked.data.gov.au/dataset/asgs2016/statesuburb/60214",
            "datatypeURI" : "http://linked.data.gov.au/def/asgs#StateSuburb",
            "datatype": "asgs16_ssc"
        }
    ]
}
```

Reply:
I'll have an update